src/fXmlRpc/Parser/NativeParser.php

```diff
@@ -26,13 +26,12 @@
 use DateTime;
 use DateTimeZone;
 use fXmlRpc\Exception\MissingExtensionException;
+use fXmlRpc\Exception\ResponseException;
```

Comment:
`ParseException` suggest that is some kind of error while parsing, but isn't, this is normal behavior of server to sending fault, maybe some kind of `FaultWrapperException`.